sass/utils/_string.scss

```diff
@@ -210,6 +210,8 @@
 /// However, it's available in all Accoutrement maps
 /// as either `'str-split'` or `'split'`.
 ///
+/// @since 4.0.2 -
+/// - New: Removed extra string length checks when `$separator` is empty
```

Comment:
I don't think we need to highlight changes like this one on the function itself, since it doesn't (we think!) impact behavior.

Reply:
Is the note in the changelog OK or should I take that away too? @mirisuzanne